build/source/engine/opSplittin.f90

```diff
@@ -251,8 +251,8 @@ subroutine opSplittin(&
  ! ------------------------------------------------------------------------------------------------------
  ! minimum timestep
  real(dp),parameter              :: dtmin_coupled=1800._dp         ! minimum time step for the fully coupled solution (seconds)
- real(dp),parameter              :: dtmin_split=360._dp            ! minimum time step for the fully split solution (seconds)
- real(dp),parameter              :: dtmin_scalar=60._dp            ! minimum time step for the scalar solution (seconds)
+ real(dp),parameter              :: dtmin_split=60._dp             ! minimum time step for the fully split solution (seconds)
+ real(dp),parameter              :: dtmin_scalar=10._dp            ! minimum time step for the scalar solution (seconds)
```

Comment:
Should we pull these out into some `constants` or `model setup` section? No change required at this time.